cmd/manager/main.go

```diff
@@ -37,15 +37,12 @@ var logLevel = flag.String("log-level", logrus.InfoLevel.String(), "Logrus log l
 func main() {
 	flag.Parse()
 
-	logLevel := logrus.InfoLevel
-
-	if parsed, err := logrus.ParseLevel(logLevel.String()); err == nil {
-		logLevel = parsed
+	parsedLevel, err := logrus.ParseLevel(*logLevel)
+	if err != nil {
+		// This should theoretically never happen
+		logrus.Errorf("log-level flag has invalid value %s", *logLevel)
```

Comment:
This seems worth a `panic(err)` or `logrus.Fatal`

Reply:
Yeah I went back and forth on this one. I think when running in standalone mode exiting makes sense here, but if we're deploying rbac manager to a cluster, exiting for an invalid log level being specified seems like something that could lead to unnecessary crash loops in the future. The worst that's going to happen here is that we're going to ignore user input and stick with the default level. If someone's trying to set the log level, they'll be looking at the logs, which makes me think this is noticeable enough. I lean towards less exits/crashes, but I can see the argument for more visible input problems.